server/swap/swap.go

```diff
@@ -34,6 +35,7 @@ var (
 	// txWaitExpiration is the longest the Swapper will wait for a coin waiter.
 	// This could be thought of as the maximum allowable backend latency.
 	txWaitExpiration = 2 * time.Minute
+	minBlockPeriod   = time.Second * 10
```

Comment:
might be useful to describe how/why this value was chosen

Reply:
maybe just some explanation regarding why this delay is needed in the first place would be nice to have (idk, one can search through git history and find this PR with the answer as well),

can suggest something like:
```
// minBlockPeriod helps with limiting notification bursts when blocks are generated too fast (e.g. in Ethereum occasionally several blocks can are generated in a matter of several seconds).
```

Reply:
Agree a sentence would be good here.

As far as the choice of 10s goes, I suppose that's short enough that it would not be an annoying delay, but also long enough that checks shouldn't pile up too much. (it works for me)